Source/Parallelization/WarpXComm_K.H

```diff
@@ -641,31 +649,25 @@ void warpx_interp (const int j,
     //     j-3   j-2   j-1    j    j+1   j+2                + c_2 * (f(j-3) + f(j+2)) / 2
     //     c_2   c_1   c_0   c_0   c_1   c_2                + ...
 
-    for (int ll = lmin; ll <= lmax; ll++)
+    for (int ll = 0; ll <= nl; ll++)
     {
 #if (AMREX_SPACEDIM == 3)
-        dl = (l - ll > 0) ? l - ll - 1 : ll - l;
```

Comment:
This `if/else` logic for choosing the index of the stencil coefficient is not necessary anymore (here and below in the loops over `k` and `j` as well), thanks to the re-ordering of the coefficients at initialization.